rust/agama-cli/src/logs.rs

```diff
@@ -0,0 +1,257 @@
+use clap::Subcommand;
+use fs_extra::copy_items;
+use fs_extra::dir::CopyOptions;
+use nix::unistd::Uid;
+use std::fs;
+use std::fs::File;
+use std::io;
+use std::io::Error;
```

Comment:
Personal taste, but I think it is better to use this error as `io::Error` instead (so you do not need to import it here). It makes possible to differentiate at first sight between `io::Error`, `format::Error` and [the Error trait](https://doc.rust-lang.org/std/error/trait.Error.html) (that I usually import as `Error`).

Reply:
it is still too much new for me to have strong opinion for such details ... however using io::Error sounds like it provides more info, so will change it.